agent/util-scripts/pbench-move-results

```diff
@@ -83,12 +99,17 @@ done
 if [[ -z "${controller}" ]]; then
     controller=${_pbench_full_hostname}
     if [[ -z "${controller}" ]]; then
-        error_log "Missing controller name (should be \"hostname -f\" value)"
+        error_log "Missing required controller name (should be \"hostname -f\" value)"
         exit 1
     fi
 fi
 validate_hostname "${controller}" "controller"
 
+if [[ -z "${user}" ]]; then
+    error_log "Missing required 'user' value (provided either by the '--user' option or the 'PBENCH_USER' environment variable)"
+    exit 1
+fi
```

Comment:
How about, "No 'PBENCH_USER' environment variable found; please specify '--user''.

Reply:
Does @dbutenhof's suggestion work?

Reply:
Yes...it's just long and less "actionable".